.circleci/orbs/helix-post-deploy/orb.yml

```diff
@@ -80,12 +87,16 @@ commands:
             description: The comma-separated list of locations to use
         newrelic_frequency: 
             type: integer
-            default: 15
+            default: 0
```

Comment:
Yes, 0 means disabled, and no, it is still an optional param with unchanged default value in the `newrelic` CLI.

I should probably remove the default value here to avoid confusion.